R/cube-query.R

```diff
@@ -1,13 +1,34 @@
 #' Crunch xtabs: Crosstab and otherwise aggregate variables in a Crunch Dataset
 #'
 #' Create a contingency table or other aggregation from cross-classifying
-#' variables in a CrunchDataset.
+#' variables in a CrunchDataset, expanding on the notation allowed in
+#' [`stats::xtabs()`] to tailor to the kinds of calculations available in crunch.
 #'
-#' @param formula an object of class 'formula' object with the
-#' cross-classifying variables separated by '+' on the right side of the
-#' "~". If aggregating by functions other than counts, include the aggregation
-#' expression on the left-hand side.
-#' Compare to [stats::xtabs()].
+#' There are 3 types of queries supported:
+#'
+#' - Crosstabs: Share the most in common with [`stats::xtabs()`], are defined by
+#' a formula with only a right hand side, with each dimension specified on the
+#' right-hand side, separated by a `+`. A dimension are generally variables, but
+#' categorical array variables contribute 2 dimensions, "categories" and "subvariables".
+#' If you just use the categorical array variable directly, the "subvariables" dimensions
+#' will be added first and the "categories" second, but you can choose their order by
```

Comment:
Probably better? How many quotes appear in our docs? [Official guidance](https://developer.r-project.org/Rds.html) is to use this stuff, and “sparingly”

```suggestion
#' categorical array variables contribute 2 dimensions, \dQuote{categories} and 
#' \dQuote{subvariables}.
#' If you just use the categorical array variable directly, the subvariables dimensions
#' will be added first and the categories second, but you can choose their order by
```